### DISCUSSION TITLE
Black video for Wan 2.1 & 2.2

### DISCUSSION THREAD

**Discussion by offroadguy56** [Haoming02/sd-webui-forge-classic#785] [Q&A] [answered]
Struggling to get Wan set up. Not many tutorials and I've only found 1 video with a proper walkthrough for Neo. Yet when followed all I get is a black video as a result.

I have tried with and without refiner. With and without OOM. Simple and SGM Uniform Scheduler. Default wan 2.2 model and a finetuned/merged 2.1 model. I am not running xformers or sage or any other performance changing parameters. 
Here's the setup:
hopping bunny
Steps: 4, Sampler: Euler, Schedule type: Simple, CFG scale: 1, Shift: 5, Seed: 1989993966, Size: 768x448, Model hash: c1d0d32561, Model: wan2114BFusionx_fusionxText2video, Clip skip: 2, RNG: CPU, Version: neo, Diffusion in Low Bits: Automatic (fp16 LoRA), Module 1: Wan2.1_VAE, Module 2: umt5-xxl-encoder-Q3_K_M

Checkpoints:
Wan2.2-Text2Video-A14B-LowNoise-Q3_K_S.gguf (quantstack higgingface)
wan2114BFusionx_fusionxText2video.safetensors (civitai)

Encoding:
umt5-xxl-encoder-Q3_K_S.gguf (both from city96 hugging face)
umt5-xxl-encoder-Q5_K_M.gguf

VAE:
Wan2.1_VAE.safetensors (for 2.1 and 2.2, everything else I believe) (from quantstack huggingface)
wan2.2_vae.safetensors (for 2.2 5B I believe) (from comfy-org huggingface)

Lora:
Wan2.2-low_noise_model.safetensors (from lightx2v / Wan2.2-Lightning)

**Comment by abzaloff** [comment]
You must use a Refiner and 2 models: high and low
https://github.com/Haoming02/sd-webui-forge-classic/wiki/Inference-References#wan-i2v

**Reply by Haoming02** [reply]
It's perfectly fine to use `LowNoise` alone *(which acts like **Wan 2.1**)*

**Comment by Haoming02** [comment]
But yes, try the exact same set up in [Inference-References](https://github.com/Haoming02/sd-webui-forge-classic/wiki/Inference-References#wan-i2v) first

**Comment by Haoming02** [comment, marked as answer]
**TL;DR:** Can't repro without more information

**Reply by offroadguy56** [reply]
> **TL;DR:** Can't repro without more information

The only thing I can think of that would break it is I'm running cuda 12.6 on a Tesla P40. If I install say version 13, I get errors in the log that features aren't supported by the card. 

I'd be happy to troubleshoot more if you have the time. 

here's the console log [https://pastebin.com/gYpCxEbX](https://pastebin.com/gYpCxEbX)

Here's nvidia SMI `Fri Feb 20 10:36:10 2026
+-----------------------------------------------------------------------------------------+
| NVIDIA-SMI 582.16                 Driver Version: 582.16         CUDA Version: 13.0     |
+-----------------------------------------+------------------------+----------------------+
| GPU  Name                  Driver-Model | Bus-Id          Disp.A | Volatile Uncorr. ECC |
| Fan  Temp   Perf          Pwr:Usage/Cap |           Memory-Usage | GPU-Util  Compute M. |
|                                         |                        |               MIG M. |
|=========================================+========================+======================|
|   0  Tesla P40                    TCC   |   00000000:00:06.0 Off |                  Off |
| N/A   41C    P0             51W /  250W |     391MiB /  24576MiB |      0%      Default |
|                                         |                        |                  N/A |
+-----------------------------------------+------------------------+----------------------+

+-----------------------------------------------------------------------------------------+
| Processes:                                                                              |
|  GPU   GI   CI              PID   Type   Process name                        GPU Memory |
|        ID   ID                                                               Usage      |
|=========================================================================================|
|    0   N/A  N/A            2008      C   ...s\Python\Python311\python.exe        234MiB |
|    0   N/A  N/A           13112      C   ...ble\python_embeded\python.exe        148MiB |
+-----------------------------------------------------------------------------------------+`

Here's extension list [https://pastebin.com/uQTVis2v](https://pastebin.com/uQTVis2v)

here's pip list `Package                   Version
------------------------- ------------------
absl-py                   2.4.0
accelerate                1.10.1
addict                    2.4.0
aiofiles                  23.2.1
aiohappyeyeballs          2.6.1
aiohttp                   3.13.3
aiosignal                 1.4.0
albucore                  0.0.24
albumentations            2.0.8
annotated-doc             0.0.4
annotated-types           0.7.0
antlr4-python3-runtime    4.9.3
anyio                     3.7.1
asttokens                 3.0.1
attrs                     25.4.0
av                        16.1.0
beautifulsoup4            4.14.3
cattrs                    25.3.0
certifi                   2026.1.4
cffi                      2.0.0
charset-normalizer        3.4.4
clean-fid                 0.1.35
click                     8.3.1
clip                      1.0
color-matcher             0.6.0
colorama                  0.4.6
contourpy                 1.3.3
cssselect2                0.8.0
cycler                    0.12.1
ddt                       1.7.2
decorator                 5.2.1
depth_anything            2024.1.22.0
depth_anything_v2         2024.7.1.0
diffusers                 0.36.0
diskcache                 5.6.3
docutils                  0.22.4
dynamicprompts            0.31.0
easydict                  1.13
einops                    0.8.1
executing                 2.2.1
facexlib                  0.3.0
fastapi                   0.112.4
ffmpeg                    1.4
ffmpy                     1.0.0
filelock                  3.20.3
filetype                  1.2.0
filterpy                  1.4.5
flatbuffers               25.12.19
fonttools                 4.61.1
freetype-py               2.5.1
frozenlist                1.8.0
fsspec                    2026.2.0
ftfy                      6.3.1
fvcore                    0.1.5.post20221221
gdown                     5.2.1
gitdb                     4.0.12
GitPython                 3.1.45
gradio                    4.40.0
gradio_client             1.2.0
gradio_imageslider        0.0.20
gradio_rangeslider        0.0.8
h11                       0.12.0
hf-xet                    1.2.0
httpcore                  0.15.0
httpx                     0.24.1
huggingface-hub           0.34.6
hydra-core                1.3.2
idna                      3.11
ImageIO                   2.37.2
imageio-ffmpeg            0.6.0
importlib_metadata        8.7.1
importlib_resources       6.5.2
inflection                0.5.1
iopath                    0.1.9
ipython                   9.10.0
ipython_pygments_lexers   1.1.1
jedi                      0.19.2
Jinja2                    3.1.6
joblib                    1.5.1
jsonmerge                 1.8.0
jsonschema                4.26.0
jsonschema-specifications 2025.9.1
kiwisolver                1.4.9
kornia                    0.6.7
lark                      1.2.2
lazy_loader               0.4
lightning                 2.5.6
lightning-utilities       0.15.2
llvmlite                  0.46.0
loadimg                   0.1.2
loguru                    0.7.3
lxml                      6.0.2
markdown-it-py            4.0.0
MarkupSafe                2.1.5
matplotlib                3.10.8
matplotlib-inline         0.2.1
mdurl                     0.1.2
mediapipe                 0.10.32
ml_dtypes                 0.5.4
mpmath                    1.3.0
multidict                 6.7.1
networkx                  3.6.1
numba                     0.63.1
numexpr                   2.14.1
numpy                     2.3.5
omegaconf                 2.2.3
onnx                      1.20.1
onnxruntime               1.24.1
open_clip_torch           2.32.0
opencv-contrib-python     4.13.0.92
opencv-python             4.8.1.78
opencv-python-headless    4.13.0.92
orjson                    3.11.7
packaging                 25.0
pandas                    2.3.3
parso                     0.8.5
peft                      0.17.1
piexif                    1.1.3
pillow                    12.0.0
pillow-avif-plugin        1.5.5
pillow_heif               1.1.1
pillow-jxl-plugin         1.3.6
PIMS                      0.6.0
pip                       26.0.1
platformdirs              4.5.1
polars                    1.38.1
polars-runtime-32         1.38.1
portalocker               3.2.0
prompt_toolkit            3.0.52
propcache                 0.4.1
protobuf                  4.25.8
psutil                    5.9.8
pure_eval                 0.2.3
pycairo                   1.29.0
pycparser                 3.0
pydantic                  2.9.2
pydantic_core             2.23.4
pydub                     0.25.1
Pygments                  2.19.2
PyMatting                 1.1.15
pyparsing                 3.3.2
PySocks                   1.7.1
python-dateutil           2.9.0.post0
python-dotenv             1.2.1
python-multipart          0.0.22
pytorch-lightning         2.6.1
pytz                      2025.2
pywin32                   311
PyYAML                    6.0.3
referencing               0.37.0
regex                     2026.1.15
reportlab                 4.4.9
requests                  2.32.5
requests-cache            1.3.0
resize-right              0.0.2
rich                      14.2.0
rlPyCairo                 0.4.0
rpds-py                   0.30.0
ruff                      0.15.0
safetensors               0.7.0
scikit-image              0.25.2
scipy                     1.17.0
seaborn                   0.13.2
segment-anything          1.0
semantic-version          2.10.0
Send2Trash                1.8.3
setuptools                69.5.1
shellingham               1.5.4
simsimd                   6.5.12
six                       1.17.0
slicerator                1.1.0
smmap                     5.0.2
sniffio                   1.3.1
sounddevice               0.5.5
soupsieve                 2.8.3
spandrel                  0.4.1
spandrel_extra_arches     0.2.0
stack-data                0.6.3
starlette                 0.38.6
stringzilla               4.6.0
svglib                    1.6.0
sympy                     1.14.0
tabulate                  0.9.0
termcolor                 3.3.0
tifffile                  2026.1.28
timm                      1.0.24
tinycss2                  1.5.1
tokenizers                0.22.2
tomesd                    0.1.3
tomlkit                   0.12.0
torch                     2.10.0+cu126
torchdiffeq               0.2.5
torchmetrics              1.8.2
torchsde                  0.2.6
torchvision               0.25.0+cu126
tqdm                      4.67.1
traitlets                 5.14.3
trampoline                0.1.2
transformers              4.56.2
transparent-background    1.3.4
triton-windows            3.5.1.post22
typer                     0.21.1
typer-slim                0.21.1
typing_extensions         4.15.0
typing-inspection         0.4.2
tzdata                    2025.3
ultralytics               8.4.12
ultralytics-thop          2.0.18
url-normalize             2.2.1
urllib3                   2.6.3
uvicorn                   0.40.0
wcwidth                   0.6.0
webencodings              0.5.1
websockets                12.0
wget                      3.2
win32_setctime            1.2.0
yacs                      0.1.8
yapf                      0.43.0
yarl                      1.22.0
zipp                      3.23.0`

**Reply by Haoming02** [reply]
Your `PasteBin` was all using `wan2114BFusionx_fusionxText2video`

Does the `.gguf` version not work either?

**Reply by offroadguy56** [reply]
negative. even with what i thought was default wan, it was failing. both the 2.1 and 2.2 versions. I've checked the link you provided and I'm downloading the comfy-org models right now. starting with fp8, if that fails i'll try the fp16 models.

**Reply by Haoming02** [reply]
Nah, don't bother with `fp16`

It's probably something else that's causing issue

**Reply by offroadguy56** [reply]
The reference page you linked seems to be the solution. I got someone sprinkling cocaine onto a pizza for "hopping bunny" prompt. Guess I need to work on the prompts. but at least I got something out of it this time. 

The 2.2VAE didn't work on the 2.2 model. So what I saw is correct, that 2.2VAE is only for 2.25B models? 

Do you know if the fp16 models would fit in 24gb vram?

**Reply by Haoming02** [reply]
Yes, the `2.2` VAE is strictly for the **5B** model. The **14B** models still use `2.1` VAE.

**Reply by Haoming02** [reply]
> Do you know if the fp16 models would fit in 24gb vram?

`fp8` version is already 14 GB, so no.

**Reply by offroadguy56** [reply]
last bit, do you know if the other Wan stuff works? I've been stuck on the T2V issue i haven't looked at the rest of it.  I just starting looking at the Vace stuff. Which can do audi to video and bunch of other inpaint like stuff.

**Reply by Haoming02** [reply]
Nah, **Forge Neo** only supports `T2V` and `I2V`

For the extra features... just use **ComfyUI** or **WanGP**
